backend/curation/api/curation-api.yml

```diff
@@ -14,6 +14,27 @@ info:
     ## Data Model
     <!-- Data model diagram created via Lucid: https://lucid.app/lucidchart/8d085832-34f4-4413-9c1d-44e77d06e326/edit?invitationId=inv_ed354532-7256-4e0a-9ad0-b622905e5198&page=Vfzkzdv~I1fL# -->
     <img src="/static/discover-api-data-model.png" width="50%"/>
+
+    ## Working with versions
+
+    #### Access to prior published version history
+    The CZ CELLxGENE Discover API employs the concept of *Versions* for our two core entities, *Collections* and
+    *Datasets*. This is to permit access to prior published instances (versions) of these entities even as new updates
+    for them are published.
+
+    #### Version identifiers
+    While the *Collection* identifier `collection_id` will always refer to the latest published instance of a Collection
+    (like the "head" in version-control terminology), and must be used with `Collection` endpoints, the
+    *Collection Version* identifier `collection_version_id` points always to one specific published instance (version)
+    of a Collection (like a commit hash) and must be used with `Collection Version` endpoints. The same is true for
+    *Datasets* and *Dataset Versions*, though it is important to note that if a given Dataset
+    is not updated during the course of a revision of its parent Collection, then its Dataset Version (and identifier
+    `dataset_version_id`) will not change even though a new Collection Version has been published. Private Collections,
+    private revisions of published Collections, and their constituent (private) Datasets are not available as Versions
+    until they are published, and therefore must be referenced via their `collection_id` and `dataset_id` identifiers
+    using `Collection` and `Dataset` endpoints. As such, mutation actions cannot be
```

Comment:
_mutation actions_ - like X-Men?

Reply:
This is the correct CRUD term I believe 😄

Reply:
@Bento007 for opinion on this